sceneformux/ux/src/main/java/com/google/ar/sceneform/ux/PlaneIntersection.java

```diff
@@ -0,0 +1,45 @@
+package com.google.ar.sceneform.ux;
+
+import androidx.annotation.Nullable;
+
+import com.google.ar.core.Plane;
+import com.google.ar.core.Pose;
+import com.google.ar.sceneform.collision.Ray;
+import com.google.ar.sceneform.math.Vector3;
+
+public class PlaneIntersection {
+    public static @Nullable
+    Pose intersect(Plane plane, Ray ray, boolean isInfinite) {
```

Comment:
inline the accessor modifiers with the declaration.

```suggestion
    @Nullable
    public static Pose intersect(Plane plane, Ray ray, boolean isInfinite) {
```